.github/labeler.yml

```diff
@@ -0,0 +1,30 @@
+# Configuration for the Labeler action
+# The action is set up in ./workflows/label.yml
+# See https://github.com/actions/labeler for more details
+#
+# Warning - Testing changes to this file can be tricky
```

Comment:
Hopefully, this is useful advice for others and clear in the advice.

Another tip I learned is how to include debug messages in the action logs (assuming the action has debug logging): https://github.com/actions/toolkit/blob/main/docs/action-debugging.md#how-to-access-step-debug-logs :)